socorro/unittest/external/test_crashstorage_base.py

```diff
@@ -155,7 +155,7 @@ def test_polyerror(self):
             )
             assert (
                 str(x)
-                == "hell,NameError('dwight',),KeyError('wilma',),AttributeError('sarita',)"
+                == "hell,NameError('dwight'),KeyError('wilma'),AttributeError('sarita')"
```

Comment:
This is comparing the str of an exception with a string. The str output changed, so I updated the expected text.